### PR TITLE
[gh-15][UI] As a user, I can search across all reports by searching-keyword

### LINKED ISSUE #15: [UI] As a user, I can search across all reports by searching-keyword
## Why

User needs to see and filter records from their previously uploaded CSV based on keyword, results, etc...

## Who Benefits?

The user who needed to use view and filter the results from uploaded CSV

### PR DESCRIPTION
**Pull request by nvminhtue**
close https://github.com/nvminhtue/elixir_internal_certificate/issues/15

## What happened

Build UI for a search keyword function
 
## Insight

- Add a text box with a search button on the top of the main page
- Add searched keyword to query string
- Adjust with current pagination href
 
## Proof Of Work
image
image

### CODE REVIEW

**Review by andyduong1920 — APPROVED** [review]

File: lib/elixir_internal_certificate_web/templates/user_search/_pagination.html.heex [hunk 1]
```diff
@@ -13,7 +13,7 @@
         <a
           class="page-link"
           aria-label="Previous"
-          href={"/keywords/?page=#{@meta.page-1}"}
+          href={"#{build_href_query(@conn, @meta.page-1)}"}
```
Comment (junan) [inline lib/elixir_internal_certificate_web/templates/user_search/_pagination.html.heex]:
```suggestion
          href={"#{build_href_query(@conn, @meta.page - 1)}"}
```

File: lib/elixir_internal_certificate_web/templates/user_search/_pagination.html.heex [hunk 2]
```diff
@@ -49,13 +49,13 @@
     <% else %>
       <li class="page-item" role="button">
         <a class="page-link" aria-label="Next"
-          href={"/keywords/?page=#{@meta.page+1}"}
+          href={"#{build_href_query(@conn, @meta.page+1)}"}
```
Comment (junan) [inline lib/elixir_internal_certificate_web/templates/user_search/_pagination.html.heex]:
```suggestion
          href={"#{build_href_query(@conn, @meta.page + 1)}"}
```

File: test/elixir_internal_certificate_web/features/home_page/search_keyword_test.exs [hunk 3]
```diff
@@ -0,0 +1,29 @@
+defmodule ElixirInternalCertificateWeb.HomePage.SearchKeyword do
+  use ElixirInternalCertificateWeb.FeatureCase, async: true
+
+  feature("search box component", %{session: session},
+    do:
```
Comment (junan) [inline test/elixir_internal_certificate_web/features/home_page/search_keyword_test.exs]: Should be do/be block as per convention

**Review by junan — APPROVED** [review]

File: test/elixir_internal_certificate_web/features/home_page/search_keyword_test.exs (hunk 3 above)
Comment (nvminhtue) [reply test/elixir_internal_certificate_web/features/home_page/search_keyword_test.exs]: Updated in [28f8882](https://github.com/nvminhtue/elixir_internal_certificate/pull/38/commits/28f8882708ab0dc7e84e09e738b1c743527875eb)

**Pull request merged**